recipes-connectivity/network-manager-applet/network-manager-applet-1.16.0/always-use-psk-hash.patch

```diff
@@ -0,0 +1,115 @@
+#OpenXT:Use WPA-PSK mechanism to authenticate on a WLAN connection.
```

Comment:
This commit (https://github.com/OpenXT/xenclient-oe/pull/1355/commits/879e60d009e2d488620fdecc039e7e6aa0ea7937) has the same message as the one backporting the upstream recipes (https://github.com/OpenXT/xenclient-oe/pull/1355/commits/4776be455030d6b9c72826132b77bb5f495c1eb0).

If I get this right:
-  https://github.com/OpenXT/xenclient-oe/pull/1355/commits/4776be455030d6b9c72826132b77bb5f495c1eb0: backports the zeus upstream recipes from upstream repositories.
- https://github.com/OpenXT/xenclient-oe/pull/1355/commits/879e60d009e2d488620fdecc039e7e6aa0ea7937: backports the OpenXT zeus patches and recipe appends.

This should be documented in the commit message for later reference.